config/jobs/periodic/docker/periodic-ci-docker.yaml

```diff
@@ -123,6 +123,47 @@ periodics:
             exit $rc
           securityContext:
             privileged: true
+    - name: periodic-integration-cli-docker
+      cluster: k8s-ppc64le-cluster
+      labels:
+              preset-build-docker: "true"
+      cron: 0 14 * * *
```

Comment:
The time set at cron clashes with the existing job `periodic-integration-test-docker`'s cron time.
Choose a time stamp at which no other job gets trigger.

Reply:
Adjusted `periodic-integration-test-docker` to 14:30 https://github.com/ppc64le-cloud/test-infra/compare/master...AshwinHIBM:test-infra:integration-cli#diff-d529e4fce74de0ef2b5f5f6f4131d73c755e5895ad4749261c3a15a2bcc6e61dR171